docs/source/intro/troubleshooting.rst

```diff
@@ -275,3 +275,51 @@ Finally, start a Jupyter server:
    $ jupyter notebook
 
 and from the newly opened browser tab select ``New -> <aiida.kernel.name>``
+
+.. _intro:increase-logging-verbosity:
+
+Increasing the logging verbosity
+--------------------------------
+
+By default, the logging level of AiiDA is minimal to avoid filling logfiles.
+Only warnings and errors are logged to the daemon log files, while info and debug
+messages are discarded.
```

Comment:
```suggestion
Only warnings and errors are logged to the daemon log files, while info and debug messages are discarded.
```
Should just apply style requirements everywhere:
 * one sentence per line
 * proper `.. code-block:: console` for `verdi` commands ..
 * .. plus prefix them with a `$` sign

Reply:
On it.